src/ClimateForcingComponent.jl

```diff
@@ -5,74 +5,71 @@
     acco2 = Parameter(index=[time])
 
     # Pre-industrial atmospheric CO2 concentration
-    co2pre = Parameter()
+    co2pre = Parameter(default = 275.0)
 
     # Atmospheric CH4 concentration
     acch4 = Parameter(index=[time])
 
     # Pre-industrial atmospheric CH4 concentration
-    ch4pre = Parameter()
+    ch4pre = Parameter(default = 790.0)
 
     # Indirect radiative forcing increase for CH4
-    ch4ind = Parameter()
+    ch4ind = Parameter(default = 0.4)
 
     # Atmospheric N2O concentration
     acn2o = Parameter(index=[time])
 
     # Pre-industrial atmospheric N2O concentration
-    n2opre = Parameter()
+    n2opre = Parameter(default = 285.0)
 
     # Atmospheric SF6 concentrations
     acsf6 = Parameter(index=[time])
 
     # Pre-industrial atmospheric SF6 concentration
-    sf6pre = Parameter()
+    sf6pre = Parameter(default = 0.04)
 
     # Radiative forcing from CO2
-    rfCO2 = Variable(index=[time])
+    rfco2 = Variable(index=[time])
 
     # Radiative forcing from CH4
-    rfCH4 = Variable(index=[time])
+    rfch4 = Variable(index=[time])
 
     # Radiative forcing from N2O
-    rfN2O = Variable(index=[time])
+    rfn2o = Variable(index=[time])
 
     # Radiative forcing from N2O
-    rfSF6 = Variable(index=[time])
+    rfsf6 = Variable(index=[time])
 
     # Radiative forcing from SO2
-    rfSO2 = Parameter(index=[time])
+    rfso2 = Parameter(index=[time])
 
     # Radiative forcing
     radforc = Variable(index=[time])
 
     # EMF22 radiative forcing
-    rfEMF22 = Variable(index=[time])
-end
-
-function interact(M, N)
-    d = 1.0 + (M * N)^0.75 * 2.01E-5 + (M * N)^1.52 * M * 5.31E-15
-    return 0.47 * log(d)
-end
+    rfemf22 = Variable(index=[time])
 
-function run_timestep(s::climateforcing, t::Int)
-    v = s.Variables
-    p = s.Parameters
-    d = s.Dimensions
+    function run_timestep(p, v, d, t)
+        
+        function interact(M, N)
+            d = 1.0 + (M * N)^0.75 * 2.01E-5 + (M * N)^1.52 * M * 5.31E-15
+            return 0.47 * log(d)
+        end
```

Comment:
Maybe define this function outside of the entire ``@defcomp`` block? I think that would be cleaner.